{"testcasedatas":[{"firstjacocotime":10.122,"tagname":"22","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase23_6_011.py","firstconsumedtime":108.101,"firstlocpath":"\\exeresult\\interestresult\\23_6_011.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage23_6_011.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase23_6_011.py\", line 137, in <module>    element = getElememtBack(driver, \"new UiSelector().text(\\\"...\\\")\", \"new UiSelector().className(\\\"android.widget.TextView\\\").instance(21)\")  File \"\\testcase\\interestcases\\testcase23_6_011.py\", line 53, in getElememtBack    element = driver.find_element_by_android_uiautomator(str2)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":55.991997,"tagname":"23","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase24_4_009.py","firstconsumedtime":131.27899,"firstlocpath":"\\exeresult\\interestresult\\24_4_009.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage24_4_009.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase24_4_009.py\", line 151, in <module>    element = getElememt(driver, \"new UiSelector().className(\\\"android.widget.ImageView\\\").description(\\\"More options\\\")\")  File \"\\testcase\\interestcases\\testcase24_4_009.py\", line 34, in getElememt    element = driver.find_element_by_android_uiautomator(str)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":50.594,"tagname":"24","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase25_4_021.py","firstconsumedtime":109.776,"firstlocpath":"\\exeresult\\interestresult\\25_4_021.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage25_4_021.ec","firsterrorlog":null},{"firstjacocotime":47.982,"tagname":"25","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase26_1_027.py","firstconsumedtime":127.998,"firstlocpath":"\\exeresult\\interestresult\\26_1_027.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage26_1_027.ec","firsterrorlog":null},{"firstjacocotime":44.239,"tagname":"26","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase27_4_009.py","firstconsumedtime":156.693,"firstlocpath":"\\exeresult\\interestresult\\27_4_009.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage27_4_009.ec","firsterrorlog":null},{"firstjacocotime":5.004,"tagname":"preference_pre_reverse","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\allpreferencecases\\precase_log.py","firstconsumedtime":182.296,"firstlocpath":"\\exeresult\\interestresult\\preference_pre_reverse.txt","firstcoveragepath":"\\coverage\\interestallcoverage\\coveragepreference_pre_reverse.ec","firsterrorlog":null},{"firstjacocotime":25.905,"tagname":"27","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase28_1_003.py","firstconsumedtime":181.268,"firstlocpath":"\\exeresult\\interestresult\\28_1_003.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage28_1_003.ec","firsterrorlog":"FAILstr(e):\t\t'NoneType' object has no attribute 'click'repr(e):\tAttributeError(\"'NoneType' object has no attribute 'click'\",)Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase28_1_003.py\", line 114, in <module>    scrollToFindElement(driver, \"new UiSelector().text(\\\"Text size\\\")\").click()AttributeError: 'NoneType' object has no attribute 'click'"},{"firstjacocotime":42.958,"tagname":"28","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase29_6_011.py","firstconsumedtime":224.176,"firstlocpath":"\\exeresult\\interestresult\\29_6_011.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage29_6_011.ec","firsterrorlog":"FAILstr(e):\t\t'NoneType' object has no attribute 'click'repr(e):\tAttributeError(\"'NoneType' object has no attribute 'click'\",)Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase29_6_011.py\", line 106, in <module>    scrollToFindElement(driver, \"new UiSelector().text(\\\"Text size\\\")\").click()AttributeError: 'NoneType' object has no attribute 'click'FAILstr(e):\t\tMessage: UiSelector[TEXT=...]repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase29_6_011.py\", line 156, in <module>    TouchAction(driver).tap(element).perform()  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\common\\touch_action.py\", line 96, in perform    self._driver.execute(Command.TOUCH_ACTION, params)  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: UiSelector[TEXT=...]"},{"firstjacocotime":55.11,"tagname":"29","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase30_6_023.py","firstconsumedtime":94.715004,"firstlocpath":"\\exeresult\\interestresult\\30_6_023.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage30_6_023.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase30_6_023.py\", line 139, in <module>    element = getElememtBack(driver, \"new UiSelector().text(\\\"...\\\")\", \"new UiSelector().className(\\\"android.widget.TextView\\\").instance(21)\")  File \"\\testcase\\interestcases\\testcase30_6_023.py\", line 53, in getElememtBack    element = driver.find_element_by_android_uiautomator(str2)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":58.234,"tagname":"30","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase31_6_011.py","firstconsumedtime":217.859,"firstlocpath":"\\exeresult\\interestresult\\31_6_011.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage31_6_011.ec","firsterrorlog":"FAILstr(e):\t\t'NoneType' object has no attribute 'click'repr(e):\tAttributeError(\"'NoneType' object has no attribute 'click'\",)Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase31_6_011.py\", line 112, in <module>    scrollToFindElement(driver, \"new UiSelector().text(\\\"Text size\\\")\").click()AttributeError: 'NoneType' object has no attribute 'click'FAILstr(e):\t\tMessage: UiSelector[TEXT=...]repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase31_6_011.py\", line 162, in <module>    TouchAction(driver).tap(element).perform()  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\common\\touch_action.py\", line 96, in perform    self._driver.execute(Command.TOUCH_ACTION, params)  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: UiSelector[TEXT=...]"},{"firstjacocotime":53.053997,"tagname":"31","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase32_6_011.py","firstconsumedtime":102.399,"firstlocpath":"\\exeresult\\interestresult\\32_6_011.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage32_6_011.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase32_6_011.py\", line 137, in <module>    element = getElememtBack(driver, \"new UiSelector().text(\\\"...\\\")\", \"new UiSelector().className(\\\"android.widget.TextView\\\").instance(21)\")  File \"\\testcase\\interestcases\\testcase32_6_011.py\", line 53, in getElememtBack    element = driver.find_element_by_android_uiautomator(str2)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":27.289,"tagname":"10","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase11_3_019.py","firstconsumedtime":84.061,"firstlocpath":"\\exeresult\\interestresult\\11_3_019.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage11_3_019.ec","firsterrorlog":null},{"firstjacocotime":27.32,"tagname":"11","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase12_4_021.py","firstconsumedtime":75.404,"firstlocpath":"\\exeresult\\interestresult\\12_4_021.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage12_4_021.ec","firsterrorlog":null},{"firstjacocotime":36.301,"tagname":"12","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase13_4_018.py","firstconsumedtime":47.253998,"firstlocpath":"\\exeresult\\interestresult\\13_4_018.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage13_4_018.ec","firsterrorlog":"FAILstr(e):\t\tMessage: TouchEvent element does not exist.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase13_4_018.py\", line 124, in <module>    TouchAction(driver).long_press(element).release().perform()  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\common\\touch_action.py\", line 96, in perform    self._driver.execute(Command.TOUCH_ACTION, params)  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: TouchEvent element does not exist."},{"firstjacocotime":5.5049996,"tagname":"13","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase14_4_001.py","firstconsumedtime":80.513,"firstlocpath":"\\exeresult\\interestresult\\14_4_001.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage14_4_001.ec","firsterrorlog":null},{"firstjacocotime":14.764,"tagname":"14","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase15_6_006.py","firstconsumedtime":67.93,"firstlocpath":"\\exeresult\\interestresult\\15_6_006.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage15_6_006.ec","firsterrorlog":null},{"firstjacocotime":41.586002,"tagname":"15","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase16_7_001.py","firstconsumedtime":108.199,"firstlocpath":"\\exeresult\\interestresult\\16_7_001.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage16_7_001.ec","firsterrorlog":"FAILstr(e):\t\tMessage: UiSelector[TEXT=...]repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase16_7_001.py\", line 151, in <module>    TouchAction(driver).long_press(element).release().perform()  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\common\\touch_action.py\", line 96, in perform    self._driver.execute(Command.TOUCH_ACTION, params)  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: UiSelector[TEXT=...]"},{"firstjacocotime":58.722,"tagname":"16","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase17_1_003.py","firstconsumedtime":69.872,"firstlocpath":"\\exeresult\\interestresult\\17_1_003.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage17_1_003.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase17_1_003.py\", line 123, in <module>    element = getElememtBack(driver, \"new UiSelector().text(\\\"...\\\")\", \"new UiSelector().className(\\\"android.widget.TextView\\\").instance(28)\")  File \"\\testcase\\interestcases\\testcase17_1_003.py\", line 53, in getElememtBack    element = driver.find_element_by_android_uiautomator(str2)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":12.61,"tagname":"17","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase18_4_009.py","firstconsumedtime":52.489,"firstlocpath":"\\exeresult\\interestresult\\18_4_009.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage18_4_009.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An unknown server-side error occurred while processing the command. Original error: The swipe did not complete successfullyrepr(e):\tWebDriverException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase18_4_009.py\", line 131, in <module>    swipe(driver, 0.5, 0.2, 0.5, 0.8)  File \"\\testcase\\interestcases\\testcase18_4_009.py\", line 65, in swipe    end_y=int(height * endyper), duration=2000)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 548, in swipe    action.perform()  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\common\\touch_action.py\", line 96, in perform    self._driver.execute(Command.TOUCH_ACTION, params)  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeWebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: The swipe did not complete successfully"},{"firstjacocotime":11.358999,"tagname":"18","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase19_1_016.py","firstconsumedtime":88.22,"firstlocpath":"\\exeresult\\interestresult\\19_1_016.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage19_1_016.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase19_1_016.py\", line 133, in <module>    element = getElememtBack(driver, \"new UiSelector().text(\\\"...\\\")\", \"new UiSelector().className(\\\"android.widget.TextView\\\").instance(21)\")  File \"\\testcase\\interestcases\\testcase19_1_016.py\", line 53, in getElememtBack    element = driver.find_element_by_android_uiautomator(str2)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":41.685997,"tagname":"19","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase20_1_016.py","firstconsumedtime":71.814,"firstlocpath":"\\exeresult\\interestresult\\20_1_016.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage20_1_016.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase20_1_016.py\", line 133, in <module>    element = getElememtBack(driver, \"new UiSelector().text(\\\"...\\\")\", \"new UiSelector().className(\\\"android.widget.TextView\\\").instance(21)\")  File \"\\testcase\\interestcases\\testcase20_1_016.py\", line 53, in getElememtBack    element = driver.find_element_by_android_uiautomator(str2)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":5.005,"tagname":"preference_pre_default","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\allpreferencecases\\defaultprecase_log.py","firstconsumedtime":232.013,"firstlocpath":"\\exeresult\\interestresult\\preference_pre_default.txt","firstcoveragepath":"\\coverage\\interestallcoverage\\coveragepreference_pre_default.ec","firsterrorlog":null},{"firstjacocotime":58.457,"tagname":"0","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase1_6_023.py","firstconsumedtime":114.615,"firstlocpath":"\\exeresult\\interestresult\\1_6_023.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage1_6_023.ec","firsterrorlog":null},{"firstjacocotime":22.698,"tagname":"1","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase2_6_012.py","firstconsumedtime":174.524,"firstlocpath":"\\exeresult\\interestresult\\2_6_012.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage2_6_012.ec","firsterrorlog":"FAILstr(e):\t\t'NoneType' object has no attribute 'click'repr(e):\tAttributeError(\"'NoneType' object has no attribute 'click'\",)Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase2_6_012.py\", line 112, in <module>    scrollToFindElement(driver, \"new UiSelector().text(\\\"Text size\\\")\").click()AttributeError: 'NoneType' object has no attribute 'click'"},{"firstjacocotime":49.725998,"tagname":"2","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase3_1_003.py","firstconsumedtime":127.522995,"firstlocpath":"\\exeresult\\interestresult\\3_1_003.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage3_1_003.ec","firsterrorlog":"FAILstr(e):\t\t'NoneType' object has no attribute 'click'repr(e):\tAttributeError(\"'NoneType' object has no attribute 'click'\",)Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase3_1_003.py\", line 112, in <module>    scrollToFindElement(driver, \"new UiSelector().text(\\\"Text size\\\")\").click()AttributeError: 'NoneType' object has no attribute 'click'FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase3_1_003.py\", line 141, in <module>    element = getElememtBack(driver, \"new UiSelector().text(\\\"...\\\")\", \"new UiSelector().className(\\\"android.widget.TextView\\\").instance(28)\")  File \"\\testcase\\interestcases\\testcase3_1_003.py\", line 53, in getElememtBack    element = driver.find_element_by_android_uiautomator(str2)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":62.489,"tagname":"3","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase4_2_012.py","firstconsumedtime":105.58099,"firstlocpath":"\\exeresult\\interestresult\\4_2_012.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage4_2_012.ec","firsterrorlog":"FAILstr(e):\t\t'NoneType' object has no attribute 'click'repr(e):\tAttributeError(\"'NoneType' object has no attribute 'click'\",)Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase4_2_012.py\", line 110, in <module>    scrollToFindElement(driver, \"new UiSelector().text(\\\"Text size\\\")\").click()AttributeError: 'NoneType' object has no attribute 'click'FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase4_2_012.py\", line 138, in <module>    element = getElememtBack(driver, \"new UiSelector().text(\\\"...\\\")\", \"new UiSelector().className(\\\"android.widget.TextView\\\").instance(19)\")  File \"\\testcase\\interestcases\\testcase4_2_012.py\", line 53, in getElememtBack    element = driver.find_element_by_android_uiautomator(str2)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":21.317,"tagname":"4","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase5_6_011.py","firstconsumedtime":112.923,"firstlocpath":"\\exeresult\\interestresult\\5_6_011.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage5_6_011.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase5_6_011.py\", line 139, in <module>    element = getElememtBack(driver, \"new UiSelector().text(\\\"...\\\")\", \"new UiSelector().className(\\\"android.widget.TextView\\\").instance(21)\")  File \"\\testcase\\interestcases\\testcase5_6_011.py\", line 53, in getElememtBack    element = driver.find_element_by_android_uiautomator(str2)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":34.425,"tagname":"5","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase6_4_009.py","firstconsumedtime":141.44101,"firstlocpath":"\\exeresult\\interestresult\\6_4_009.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage6_4_009.ec","firsterrorlog":null},{"firstjacocotime":28.224,"tagname":"6","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase7_1_011.py","firstconsumedtime":111.724,"firstlocpath":"\\exeresult\\interestresult\\7_1_011.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage7_1_011.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An element could not be located on the page using the given search parameters.repr(e):\tNoSuchElementException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase7_1_011.py\", line 131, in <module>    element = getElememt(driver, \"new UiSelector().className(\\\"android.widget.TextView\\\").instance(32)\")  File \"\\testcase\\interestcases\\testcase7_1_011.py\", line 34, in getElememt    element = driver.find_element_by_android_uiautomator(str)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 327, in find_element_by_android_uiautomator    return self.find_element(by=By.ANDROID_UIAUTOMATOR, value=uia_string)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 219, in find_element    'value': value})['value']  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeNoSuchElementException: Message: An element could not be located on the page using the given search parameters."},{"firstjacocotime":60.728,"tagname":"7","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase8_4_029.py","firstconsumedtime":63.713997,"firstlocpath":"\\exeresult\\interestresult\\8_4_029.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage8_4_029.ec","firsterrorlog":null},{"firstjacocotime":60.77,"tagname":"8","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase9_1_016.py","firstconsumedtime":41.056,"firstlocpath":"\\exeresult\\interestresult\\9_1_016.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage9_1_016.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An unknown server-side error occurred while processing the command. Original error: The swipe did not complete successfullyrepr(e):\tWebDriverException()Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase9_1_016.py\", line 131, in <module>    swipe(driver, 0.5, 0.2, 0.5, 0.8)  File \"\\testcase\\interestcases\\testcase9_1_016.py\", line 65, in swipe    end_y=int(height * endyper), duration=2000)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 548, in swipe    action.perform()  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\common\\touch_action.py\", line 96, in perform    self._driver.execute(Command.TOUCH_ACTION, params)  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeWebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: The swipe did not complete successfully"},{"firstjacocotime":12.558,"tagname":"9","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\interestcases\\testcase10_1_027.py","firstconsumedtime":86.266,"firstlocpath":"\\exeresult\\interestresult\\10_1_027.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage10_1_027.ec","firsterrorlog":null},{"firstjacocotime":57.588,"tagname":"20","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase21_6_023.py","firstconsumedtime":115.852005,"firstlocpath":"\\exeresult\\interestresult\\21_6_023.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage21_6_023.ec","firsterrorlog":"FAILstr(e):\t\t'NoneType' object has no attribute 'click'repr(e):\tAttributeError(\"'NoneType' object has no attribute 'click'\",)Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase21_6_023.py\", line 112, in <module>    scrollToFindElement(driver, \"new UiSelector().text(\\\"Text size\\\")\").click()AttributeError: 'NoneType' object has no attribute 'click'"},{"firstjacocotime":61.386,"tagname":"21","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\interestcases\\testcase22_6_023.py","firstconsumedtime":130.445,"firstlocpath":"\\exeresult\\interestresult\\22_6_023.txt","firstcoveragepath":"\\coverage\\interestcoverage\\coverage22_6_023.ec","firsterrorlog":"FAILstr(e):\t\t'NoneType' object has no attribute 'click'repr(e):\tAttributeError(\"'NoneType' object has no attribute 'click'\",)Traceback (most recent call last):  File \"\\testcase\\interestcases\\testcase22_6_023.py\", line 112, in <module>    scrollToFindElement(driver, \"new UiSelector().text(\\\"Text size\\\")\").click()AttributeError: 'NoneType' object has no attribute 'click'"}],"interestmap":[]}